src/steps/video-setup/prefs.tsx

```diff
@@ -434,11 +441,12 @@ type RadioButtonProps = {
   checked: boolean;
   label?: string;
   onChange: (v: string) => void;
+  isHighContrast: boolean;
 };
 
 // A styled radio input which looks like a button.
 const RadioButton: React.FC<RadioButtonProps> = ({
-  id, value, checked, name, onChange, label,
+  id, value, checked, name, onChange, label, isHighContrast,
 }) => {
   return <div>
```

Comment:
Again, `RadioButton` can easily use `useColorScheme()` itself, so no need to pass it as prop. 
You did this in some places where the component didn't have a block body, but immediately returned something. There I can understand that it's annoying to re-indent the whole body just to add a hook. But here one easily can.